### PR TITLE
Add inactive robot disassembly recipe

### LINKED ISSUE #36883: Some inactive bots can't be disassembled
# Describe the bug

Some robots can have their corpses disassembled, but not their deactivated item form.

# Steps To Reproduce

1. Kill a riot control platform, riot dispatch, or military dispatch
2. Disassemble the broken riot control turret or broken dispatch
3. Get a control laptop and hack a riot control platform or dispatch
4. Move into and deactivate it
5. The inactive riot control turret or inactive dispatch can't be disassembled

# Expected behavior

Inactive robots can be disassembled into loot that is as good as, or better than, the loot from disassembling a broken robot of the same type. 

# Screenshots

None

# Versions and configuration

- OS: MacOs
    - OS Version: Name: Mac OS X; Version: 10.14.6; Build: 18G2022; 
- Game Version: 0.D-11103-gbd3a2e3 [64-bit]
- Graphics Version: Tiles
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food]
]

# Additional context

Inactive broken/prototype cyborgs also can't be disassembled, but they probably shouldn't. Maybe a psychopath character could butcher them.

### PR DESCRIPTION
**Pull request by DziugasPraninskas**
#### Summary

`SUMMARY: Content "Adds inactive robot disassembly recipes '"`
#### Purpose of change

Fixes #36883.
#### Describe the solution

Add disassembly recipes to inactive robots.
#### Describe alternatives you've considered

Currently I kept the material amounts the same as the broken robots, but made it slightly easier and faster to disassemble.
#### Testing

1. Loaded up the game.
2. Disassembled inactive robot.

### CODE REVIEW

**Comment by Fris0uman** [comment]
For inactive robot you should add crafting recipes and make them reversible.
So those recipe should go in recipe_electronics.json with the other inactive robots. If you look at the robots already there they took more times that the broken ones, you also need to add the proper schematics. See how https://github.com/CleverRaven/Cataclysm-DDA/pull/33266 and https://github.com/CleverRaven/Cataclysm-DDA/pull/30456 did it.

**Comment by Night-Pryanik** [comment]
@Dziugaske is this PR ready?

**Comment by Dziugaske** [comment]
@Night-Pryanik Sorry for the late reply. Yes, the PR is ready.

File: data/json/recipes/recipe_electronics.json [hunk 1]
```diff
@@ -1265,7 +1265,7 @@
     "skill_used": "electronics",
     "skills_required": [ [ "mechanics", 6 ], [ "computer", 6 ], [ "fabrication", 6 ] ],
     "difficulty": 7,
-    "time": "6 h",
+    "time": "30 m",
```
Comment (ZhilkinSerg) [inline data/json/recipes/recipe_electronics.json:1268]:
```suggestion
    "time": "6 h",
```

**Pull request merged**